### ISSUE TITLE
source and wheel versions of 1.0.0 have different migrations

### ISSUE THREAD

**Issue by jgoettsch** [stefanfoulis/django-filer#602]
You can verify this by downloading and inspecting [the source from PyPI](https://pypi.python.org/pypi/django-filer/1.0.0) or by installing with different versions of pip (i.e. an older version to install from source, a newer version to install the wheel).

In the source version I see:

```
$ ls -lR
total 0
drwxr-xr-x   5 jeff  staff  170 Nov  3 13:06 migrations/
drwxr-xr-x  17 jeff  staff  578 Nov  3 13:06 south_migrations/

./migrations:
total 32
-rw-r--r--  1 jeff  staff  9539 Nov  3 09:27 0001_initial.py
-rw-r--r--  1 jeff  staff   521 Nov  3 09:27 0002_auto_20150606_2003.py
-rw-r--r--  1 jeff  staff     0 Oct 14  2014 __init__.py

./south_migrations:
total 344
-rw-r--r--  1 jeff  staff  16347 Nov  3 09:27 0001_initial.py
-rw-r--r--  1 jeff  staff   9155 Nov  3 09:27 0002_rename_file_field.py
-rw-r--r--  1 jeff  staff   9468 Nov  3 09:27 0003_add_description_field.py
-rw-r--r--  1 jeff  staff  11108 Nov  3 09:27 0004_auto__del_field_file__file__add_field_file_file__add_field_file_is_pub.py
-rw-r--r--  1 jeff  staff  10525 Nov  3 09:27 0005_auto__add_field_file_sha1__chg_field_file_file.py
-rw-r--r--  1 jeff  staff  10531 Nov  3 09:27 0006_polymorphic__add_field_file_polymorphic_ctype.py
-rw-r--r--  1 jeff  staff  11273 Nov  3 09:27 0007_polymorphic__content_type_data.py
-rw-r--r--  1 jeff  staff  10351 Nov  3 09:27 0008_polymorphic__del_field_file__file_type_plugin_name.py
-rw-r--r--  1 jeff  staff  11635 Nov  3 09:27 0009_auto__add_field_folderpermission_can_edit_new__add_field_folderpermiss.py
-rw-r--r--  1 jeff  staff  11223 Nov  3 09:27 0010_folderpermissions.py
-rw-r--r--  1 jeff  staff  11222 Nov  3 09:27 0011_auto__del_field_folderpermission_can_add_children__del_field_folderper.py
-rw-r--r--  1 jeff  staff  10617 Nov  3 09:27 0012_renaming_folderpermissions.py
-rw-r--r--  1 jeff  staff  11402 Nov  3 09:27 0013_remove_null_file_name.py
-rw-r--r--  1 jeff  staff  11357 Nov  3 09:27 0014_auto__add_field_image_related_url__chg_field_file_name.py
-rw-r--r--  1 jeff  staff      0 Nov  3 09:27 __init__.py
```

and in the wheel version I see:

```
$ ls -lR
total 0
drwxr-xr-x  18 jeff  staff  612 Nov  4 16:16 migrations/
drwxr-xr-x   4 jeff  staff  136 Nov  4 16:16 migrations_django/
drwxr-xr-x  17 jeff  staff  578 Nov  4 16:16 south_migrations/

./migrations:
total 344
-rw-r--r--  1 jeff  staff   9539 Nov  3 12:27 0001_initial.py
-rw-r--r--  1 jeff  staff    521 Nov  3 12:27 0002_auto_20150606_2003.py
-rw-r--r--  1 jeff  staff   9155 Oct 14  2014 0002_rename_file_field.py
-rw-r--r--  1 jeff  staff   9468 Oct 14  2014 0003_add_description_field.py
-rw-r--r--  1 jeff  staff  11108 Oct 14  2014 0004_auto__del_field_file__file__add_field_file_file__add_field_file_is_pub.py
-rw-r--r--  1 jeff  staff  10525 Oct 14  2014 0005_auto__add_field_file_sha1__chg_field_file_file.py
-rw-r--r--  1 jeff  staff  10531 Oct 14  2014 0006_polymorphic__add_field_file_polymorphic_ctype.py
-rw-r--r--  1 jeff  staff  11273 Oct 14  2014 0007_polymorphic__content_type_data.py
-rw-r--r--  1 jeff  staff  10351 Oct 14  2014 0008_polymorphic__del_field_file__file_type_plugin_name.py
-rw-r--r--  1 jeff  staff  11635 Oct 14  2014 0009_auto__add_field_folderpermission_can_edit_new__add_field_folderpermiss.py
-rw-r--r--  1 jeff  staff  11223 Oct 14  2014 0010_folderpermissions.py
-rw-r--r--  1 jeff  staff  11222 Oct 14  2014 0011_auto__del_field_folderpermission_can_add_children__del_field_folderper.py
-rw-r--r--  1 jeff  staff  10617 Oct 14  2014 0012_renaming_folderpermissions.py
-rw-r--r--  1 jeff  staff  11402 Oct 14  2014 0013_remove_null_file_name.py
-rw-r--r--  1 jeff  staff  11357 Oct 14  2014 0014_auto__add_field_image_related_url__chg_field_file_name.py
-rw-r--r--  1 jeff  staff      0 Oct 14  2014 __init__.py

./migrations_django:
total 24
-rw-r--r--  1 jeff  staff  9348 Oct 14  2014 0001_initial.py
-rw-r--r--  1 jeff  staff     0 Oct 14  2014 __init__.py

./south_migrations:
total 344
-rw-r--r--  1 jeff  staff  16347 Nov  3 12:27 0001_initial.py
-rw-r--r--  1 jeff  staff   9155 Nov  3 12:27 0002_rename_file_field.py
-rw-r--r--  1 jeff  staff   9468 Nov  3 12:27 0003_add_description_field.py
-rw-r--r--  1 jeff  staff  11108 Nov  3 12:27 0004_auto__del_field_file__file__add_field_file_file__add_field_file_is_pub.py
-rw-r--r--  1 jeff  staff  10525 Nov  3 12:27 0005_auto__add_field_file_sha1__chg_field_file_file.py
-rw-r--r--  1 jeff  staff  10531 Nov  3 12:27 0006_polymorphic__add_field_file_polymorphic_ctype.py
-rw-r--r--  1 jeff  staff  11273 Nov  3 12:27 0007_polymorphic__content_type_data.py
-rw-r--r--  1 jeff  staff  10351 Nov  3 12:27 0008_polymorphic__del_field_file__file_type_plugin_name.py
-rw-r--r--  1 jeff  staff  11635 Nov  3 12:27 0009_auto__add_field_folderpermission_can_edit_new__add_field_folderpermiss.py
-rw-r--r--  1 jeff  staff  11223 Nov  3 12:27 0010_folderpermissions.py
-rw-r--r--  1 jeff  staff  11222 Nov  3 12:27 0011_auto__del_field_folderpermission_can_add_children__del_field_folderper.py
-rw-r--r--  1 jeff  staff  10617 Nov  3 12:27 0012_renaming_folderpermissions.py
-rw-r--r--  1 jeff  staff  11402 Nov  3 12:27 0013_remove_null_file_name.py
-rw-r--r--  1 jeff  staff  11357 Nov  3 12:27 0014_auto__add_field_image_related_url__chg_field_file_name.py
-rw-r--r--  1 jeff  staff      0 Nov  3 12:27 __init__.py
```

**Comment by mkoistinen** [comment]
That is really strange. Will look into it.

**Comment by mkoistinen** [comment]
Fixed in v1.0.1

**Comment by jokerejoker** [comment]
Django migrations, in 1.0.1, is now placed in migrations_django package and south migrations are placed in migrations package.

**Comment by yakky** [comment]
@jokerejoker this should have been fixed in 1.0.2

**Comment by jokerejoker** [comment]
@yakky I can confirm the fix for 1.0.2, now also for the wheel package ;P

**Comment by yakky** [comment]
Thanks @jokerejoker for checking!
